src/omt/bitvector_optimizer.cpp

```diff
@@ -33,20 +33,19 @@ BitVector OMTOptimizerBitVector::computeAverage(const BitVector& a,
   // computes (a + b) / 2 without overflow
   // rounding towards -infinity: -1.5 --> -2,  1.5 --> 1
   // average = (a / 2) + (b / 2) + (((a % 2) + (b % 2)) / 2)
+  // For unsigned bitvectors, this algorithm should be equivalent to:
+  // if (!isSigned) {
+  //   return BitVector(a.getSize(),
+  //     (a.toInteger() + b.toInteger()).floorDivideQuotient(Integer(2)));
+  // }
```

Comment:
Since this is a comment, I suggest using ordinary english/math. Otherwise it looks like commented out code.

Reply:
not this time. the test will be added after everything is up.

Reply:
and the comments could be removed.

Reply:
this piece of code-like comment is removed. A narrative description is already included in the function.